staging/src/k8s.io/kubectl/pkg/cmd/create/create_token.go

```diff
@@ -208,7 +208,7 @@ func (o *TokenOptions) Validate() error {
 	if len(o.Namespace) == 0 {
 		return fmt.Errorf("--namespace is required")
 	}
-	if o.Duration < 0 || (o.Duration == 0 && o.Flags.Changed("duration")) {
+	if o.Duration < 0 {
 		return fmt.Errorf("--duration must be positive")
```

Comment:
Change this to `--duration must be greater than or equal to 0` for accuracy

Reply:
Good point. Updated, please take a look.